plugins/cue_points/quiz/sphinx/AnswerCuePointSphinxPlugin.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Enable indexing and searching answers cue point objects in sphinx
+ * @package plugins.cuePoint
+ */
+class AnswerCuePointSphinxPlugin extends KalturaPlugin implements IKalturaCriteriaFactory
```

Comment:
1. change plugin name to QuizSphinxPlugin
2. implement IKalturaPending interface and add dependency to CuePoint plugin and Quiz plugin